update_images

```diff
@@ -19,18 +19,18 @@ while read -r line; do
 
     if [[ "${os}" = 'debian' ]] || [[ "${os}" = 'ubuntu' ]]; then
         tag="${os}-${release}-all"
-        args+="build -t citusdata/packaging:${tag} -f ${dockerfiles_dir}/${tag}/Dockerfile .\n"
+        args+="build --rm=false -t citus/packaging:${tag} -f ${dockerfiles_dir}/${tag}/Dockerfile .\n"
```

Comment:
Can you explain the changes on this line? I don't understand why they're necessary.

Reply:
This change helped me recover faster during development. I recently reverted this change.